velox/exec/HashAggregation.cpp

```diff
@@ -51,17 +51,9 @@ HashAggregation::HashAggregation(
 
   auto inputType = aggregationNode->sources()[0]->outputType();
 
-  auto numHashers = aggregationNode->groupingKeys().size();
-  std::vector<std::unique_ptr<VectorHasher>> hashers;
-  hashers.reserve(numHashers);
-  for (const auto& key : aggregationNode->groupingKeys()) {
-    auto channel = exprToChannel(key.get(), inputType);
-    VELOX_CHECK_NE(
```

Comment:
We are losing this check

Reply:
This check was redundant to begin with. exprToChannel never returns constant channel for a field reference.